python/tank/bootstrap/manager.py

```diff
@@ -541,6 +542,7 @@ def failed_callback(phase, exception):
         :type entity: Dictionary with keys ``type`` and ``id``, or ``None`` for the site.
         :param completed_callback: Callback function that handles cleanup after successful completion of the bootstrap.
         :param failed_callback: Callback function that handles cleanup after failed completion of the bootstrap.
+        :param parent: The parent object used for the async bootstrapper.
```

Comment:
Please give some background here to explain when a parent is needed and when it can be left as None. can we provide an example of what this typically should look like? (e.g find main application and parent under there, or something like that). I feel if we just say "the parent object used for the async bootstrapper" it's not really telling me much more than i already knew from just reading the parameter name "parent".

Reply:
👍